scripts/smoke.sh

```diff
@@ -13,8 +13,9 @@ source "${PROGDIR}/.util/tools.sh"
 source "${PROGDIR}/.util/print.sh"
 
 function main() {
-  local name token
+  local name token regport
   token=""
+  regport=""
```

Comment:
Not a blocker, but the next time we touch this file it would be good to clean this up slightly by setting the default here (i.e. `regport="5000"`) and removing the `if -z` later on. Also I'd add `(optional)` to the help text.

Reply:
Sounds sensible...  I was trying to follow the existing logic for `name` that has an `if -z` block. Should that also be set as a default at declaration ? (And should name also be documented as optional in the help text?)

Reply:
Oh, I didn't see `name`. Yes, I'd apply the same logic to both `name` and `regport` - default values when declaring/defining the variable and and `(optional)` in the help text.